ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -1220,6 +1220,37 @@ def test_torch_instance_amin(
     )
 
 
+# aminmax
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="aminmax",
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric", full=True),
+    ),
+)
+def test_torch_instance_aminmax(
+    dtype_x,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+):
+    input_dtype, x = dtype_x
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtype,
+        init_all_as_kwargs_np={
+            "data": x[0],
+        },
+        method_input_dtypes=input_dtype,
+        method_all_as_kwargs_np={},
```

Comment:
We should also generate a flag `keepdim` using `st.booleans()` strategy and pass it as a method parameter

Reply:
Got it! I had thought that flag had been removed in the refactor, thanks! Added it back :)